style.css

```diff
@@ -0,0 +1,11 @@
+body {
+  background-color: powderblue;
+}
+
+h1 {
+  color: blue;
+}
+
+p {
+  color: red;
+}
```

Comment:
It will be good to fix all the linter errors. I can see some stylelint is failing. This tells me that there are some linting errors in your styles. Fixing this will make your project professional and the linters helps you follow best practices. Kindly try solve all the linter errors